tabs/pid_tuning.js

```diff
@@ -499,7 +499,8 @@ TABS.pid_tuning.initialize = function (callback) {
         context.restore();
     }
 
-    function process_html() {
+    function process_html() 
+    {
```

Comment:
I wouldn't go for K&R-ish braces in javascript. Since javascript has first class functions this does not make much sense.